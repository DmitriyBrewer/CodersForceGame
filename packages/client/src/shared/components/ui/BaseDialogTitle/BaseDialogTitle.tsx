import { DialogTitle } from '@mui/material'

const BaseDialogTitle = DialogTitle

export default BaseDialogTitle
