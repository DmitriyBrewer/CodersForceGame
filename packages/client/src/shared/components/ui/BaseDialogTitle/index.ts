import BaseDialogTitle from './BaseDialogTitle'

export default BaseDialogTitle
