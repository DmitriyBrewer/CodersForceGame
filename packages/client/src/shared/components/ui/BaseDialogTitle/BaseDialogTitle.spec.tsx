import { render, screen } from '@testing-library/react'

import '@testing-library/jest-dom'
import BaseDialogTitle from './BaseDialogTitle'

describe('<BaseDialogTitle />', () => {
  const handleClick = jest.fn()

  it('Snapshot is correct', () => {
    const { asFragment } = render(<BaseDialogTitle onClick={handleClick} />)
    expect(asFragment()).toMatchSnapshot()
  })

  it('Renders correctly with default props', () => {
    render(<BaseDialogTitle />)
    const button = screen.getByRole('button')

    expect(button).toMatchSnapshot()
  })
})
