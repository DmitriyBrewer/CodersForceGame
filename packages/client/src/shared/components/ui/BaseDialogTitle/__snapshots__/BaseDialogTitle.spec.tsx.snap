// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BaseDialogTitle /> Renders correctly with default props 1`] = `
<h2
  class="MuiTypography-root MuiTypography-h6 MuiDialogTitle-root css-jsgecs-MuiTypography-root-MuiDialogTitle-root"
/>
`;

exports[`<BaseDialogTitle /> Snapshot is correct 1`] = `
<DocumentFragment>
  <h2
    class="MuiTypography-root MuiTypography-h6 MuiDialogTitle-root css-jsgecs-MuiTypography-root-MuiDialogTitle-root"
  />
</DocumentFragment>
`;
